deploy/olm-catalog/host-operator/host-operator.package.yaml

```diff
@@ -0,0 +1,5 @@
+channels:
```

Comment:
There should not be any package yamls

Reply:
I've removed this yaml file, but I'd like to know why it is getting generated in the first place.  I've removed my vendor dir and run `make clean` in the api project, so I'm not sure where this is coming from.

Reply:
That's weird. Are you sure that you use the latest versions of both projects? What target do you use for generating and propagating the CRD changes in api project? You should use `make generate` - that should do everything necessary for you including creating/updating the correct CSV and operator package